doc/build-netbsd.md

```diff
@@ -1,81 +1,161 @@
-NetBSD build guide
-======================
-(updated for NetBSD 8.0)
+# NetBSD Build Guide
 
-This guide describes how to build bitcoind and command-line utilities on NetBSD.
+**Updated for NetBSD [9.1](https://netbsd.org/releases/formal-9/NetBSD-9.1.html)**
```

Comment:
nit, I think the bold is unneeded here and for "required" and "optional" below. In general, I'd go light on italic and bold.